deploy/vagrant/provision/roles/ufs_hadoop1/files/config_tachyon.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 
-sed -i "s/^export TACHYON_UNDERFS_ADDRESS=.*/export TACHYON_UNDERFS_ADDRESS=hdfs:\/\/\$\{TACHYON_MASTER_ADDRESS\}:9000/g" /tachyon/conf/tachyon-env.sh
+UFS_MASTER=$(tail -n1 /tachyon/conf/workers)
```

Comment:
Yes. Actually, it is always `TachyonMaster` now, but in case we will change the name to `TachyonMaster1`, I make the assumption that it is always the last in conf/workers.